sdk/tables/azure-data-tables/samples/README.md

```diff
@@ -11,7 +11,7 @@ urlFragment: tables-samples
 # Samples for Azure Tables client library for Python
 
 These code samples show common scenario operations with the Azure Data Tables client library.
-The async versions of the samples require Python 3.5 or later.
+The async versions of the samples require Python 3.7 or later.
```

Comment:
Since the SDK itself do not support below 3.7 now, this entire sentence can be removed